src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -17,6 +22,7 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
 
   public constructor(
     private readonly myWalletsService: PagesWalletsMyWalletsService,
+    public dialog: MatDialog,
```

Comment:
Why dialog is `public`? :) Remember to use `public` only if you are using something in the template `html file`.